### DISCUSSION TITLE
Using custom fonts

### DISCUSSION THREAD

**Discussion by ubiuser** [gotenberg/gotenberg#423] [Q&A] [answered]
How should I refer to the custom fonts in CSS that I add to the Docker image?
Docker file:
```
FROM gotenberg/gotenberg:7

USER root

COPY my-custom-font.woff /usr/local/share/fonts/

RUN usermod -u 1001 -g 1001 gotenberg

USER gotenberg
```

CSS, where basically `<reference-to-my-custom-font.woff>` is the question:
```
@font-face {
  font-family: MyCustomFont;
  src: url(<reference-to-my-custom-font.woff>) format("woff");
}
```

Also, do I need to set `--chromium-allow-file-access-from-files=true`?

Many thanks!

**Comment by gulien** [comment, marked as answer]
Hello @ubiuser,

You actually don't need `--chromium-allow-file-access-from-files`: it's when a script requires for instance to read other local files (via `file://`).

I also don't think you have to specify `src: url(<reference-to-my-custom-font.woff>) format("woff");`: just use the correct `font-family` 😄
